ts/features/wallet/cobadge/component/CobadgeWalletPreview.tsx

```diff
@@ -62,18 +67,22 @@ const CobadgeWalletPreview: React.FunctionComponent<Props> = props => {
     BASE_IMG_H,
     props.cobadge.abiInfo?.logoUrl
   );
+
+  const brandLogo = getCardIconFromBrandLogo(props.cobadge.info);
   return (
     <CardPreview
       left={renderLeft(props, imgDimensions)}
-      image={getCardIconFromBrandLogo(props.cobadge.info)}
-      onPress={() => props.navigateToCobadgeDetails(props.cobadge)}
+      image={brandLogo}
+      onPress={() => props.navigateToCobadgeDetails(props.cobadge, brandLogo)}
```

Comment:
What you think about passing only `props.cobadge` as navigation param? The brandlogo can always be uniquely derived using `getCardIconFromBrandLogo`.

Reply:
Yes, sure.
Modified in [0816b1d](https://github.com/pagopa/io-app/pull/2751/commits/0816b1d1cca33008d4900f323e424274e7f2fb49)